docs/e2e.md

```diff
@@ -0,0 +1,152 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# End to End
+
+RamenDR end-to-end (e2e) tests validate various scenarios for regional DR
+using predefined workloads and deployment methods.
+
+## Prepare `config.yaml` File
+
+Create a `config.yaml` file by copying the `config.yaml.sample` template:
+
+```sh
+cat e2e/config.yaml.sample >> e2e/config.yaml
+```
+
+Update `config.yaml` by adding kubeconfig paths of the clusters, with format:
+
+```yaml
+Clusters:
+    c1:
+        kubeconfigpath: /path/to/kubeconfig/c1
+    c2:
+        kubeconfigpath: /path/to/kubeconfig/c2
+    hub:
+        kubeconfigpath: /path/to/kubeconfig/hub
+```
+
+If `drenv` was used to configure RDR clusters, easily copy `config.yaml.sample`
+and add the clusters kubeconfig paths into `config.yaml` using:
+
+```sh
+cat e2e/config.yaml.sample ~/.config/drenv/rdr/config.yaml >> e2e/config.yaml
```

Comment:
We don't need to append, this will create broken config on the second run.

```
cat e2e/config.yaml.sample ~/.config/drenv/rdr/config.yaml > e2e/config.yaml
```

This is the single command that every developer will use a lot, so maybe it should be the first command displayed in this section.

We can have this structure:

- Preparing config.yaml
  - For drenv environemnt
  - For real cluster

For drenv we have single command. For real cluster need to edit the file manually so showing clusters yaml forma is useful.